src/main/webapp/app/overview/course-conversations/layout/conversation-messages/conversation-messages.component.ts

```diff
@@ -301,7 +306,80 @@ export class ConversationMessagesComponent implements OnInit, AfterViewInit, OnD
                 return post;
             });
 
-        this.groupPosts();
+        const postIdsWithForwardedMessages = this.posts.filter((post) => post.hasForwardedMessages && post.id !== undefined).map((post) => post.id) as number[];
+
+        if (postIdsWithForwardedMessages.length > 0) {
```

Comment:
Maybe we could invert some of these ifs to reduce a bit of the nesting.

Reply:
I think this increases readability for now so I left it as it is